src/pages/VenueLandingPage/VenueLandingPageContent.tsx

```diff
@@ -87,7 +95,7 @@ const VenueLandingPageContent: React.FC<VenueLandingPageContentProps> = ({
   const containerClasses = classNames("header", containerVars);
 
   return (
-    <div className="container venue-entrance-experience-container">
+    <div className="VenueLandingPageContent container venue-entrance-experience-container">
```

Comment:
this `container` doesn't seem to be used anywhere. might as well remove it

Reply:
Deliberate moniker. It isn't the only measure to be used in a SCSS file. This one makes the top element of the component visible in the browser inspector and hopefully some day make it easier to have custom styles by users.

So, my rule of thumb is:
- Always put the component name as a class to the top level element in it

Reply:
The other ones.... didn't want to get into a whole cleaning process, wanted to just have it most as it is.